### ISSUE TITLE
Work out what to do with dancing penguins

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#311]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/penguinmark/

They're currently made out of a bunch of empty divs with CSS bg images, which can't have alt text. Changing to images likely changes perf characteristics too much. Also need to work out what is appropriate to report to screen reader.

Possible solution:

* add aria-label to the top level div of each penguin labelling what it is doing "Opera penguin dancing" or whatever. 

There are probably other badly labelled things on this page, like the tree looks like it has no alt text either, nor alt='".

May be best to add hidden text describing what the page is doing for screen reader users.
